### ISSUE TITLE
parsing error of config.yml results in feedback "Unable to find a configuration file"

### ISSUE THREAD

**Issue by barzilay** [spriteCloud/lapis-lazuli#81]
**Actual**

- When there is a parse error in your config.yml configuration file, you will get the following feedback when you start a test:
`Unable to find a configuration file, defaulting to empty config.yml.`
- The test run continues

**Expected**
- Throw the parsing error
- Stop the test

**Comment by sjieg** [comment]
While working on this one, it might be nice to extend the error reporting as following:

**Current**
```
  Unknown environment variable 'acceptance.pages.root.url' and no default given
```

**Possible improvement**
```
The environment variable 'acceptance.pages.root.url' was not found:
- acceptance > exists
- acceptance.pages > exists
- acceptance.pages.root > does not exist
```

@barzilay WDYT?

**Comment by barzilay** [comment]
Expect the following feedback (or similar) when parsing fails for a config file:

`2.3.1 :006 > x = YAML.load_file('config.yml')

Psych::SyntaxError: (config.yml): mapping values are not allowed in this context at line 51 column 16
	from /Users/Barzilay/.rvm/gems/ruby-2.3.1/gems/psych-2.2.4/lib/psych.rb:377:in `parse'
	from /Users/Barzilay/.rvm/gems/ruby-2.3.1/gems/psych-2.2.4/lib/psych.rb:377:in `parse_stream'
	from /Users/Barzilay/.rvm/gems/ruby-2.3.1/gems/psych-2.2.4/lib/psych.rb:325:in `parse'
	from /Users/Barzilay/.rvm/gems/ruby-2.3.1/gems/psych-2.2.4/lib/psych.rb:252:in `load'
	from /Users/Barzilay/.rvm/gems/ruby-2.3.1/gems/psych-2.2.4/lib/psych.rb:473:in `block in load_file'
	from /Users/Barzilay/.rvm/gems/ruby-2.3.1/gems/psych-2.2.4/lib/psych.rb:472:in `open'
	from /Users/Barzilay/.rvm/gems/ruby-2.3.1/gems/psych-2.2.4/lib/psych.rb:472:in `load_file'
	from (irb):6
	from /Users/Barzilay/.rvm/rubies/ruby-2.3.1/bin/irb:11:in `<main>'`

**Comment by barzilay** [comment]
@sjieg, "Possible improvement" looks like a nice to have. When the config parsing and feedback is correct I expect though that its much clearer how to configure and call variables so lets stick with only the fix for this ticket.

**Comment by mauricewijnia** [comment]
Made a pull request that fixes this issue

**Comment by barzilay** [comment]
Fixed in [LL 2.0.1](https://rubygems.org/gems/lapis_lazuli/versions/2.0.1)
